jaystaks/Politico
# Ch 163884001 fix tests tdd way

### What does this PR do?
Test the  endpoints using TDD approach

### Description of Task to be completed?

   1.  Set the test for the endpoints in the tests module

### How should this be manually tested?
Go to https://github.com/jaystaks/Politico, make sure you are on the develop branch, in the readMe file there is a travis build badge, click on it to view the build for the test

What are the relevant pivotal tracker stories?
#163884001
